app/templates/devices.html

```diff
@@ -54,7 +54,7 @@ <h2 id="m_{{machine_type}}">Picobrew S/Pro</h2>
       {% else %}
       <h2 id="m_{{machine_type}}">{{machine_type}}</h2>
       {% endif %}
-      {% if config['aliases'][machine_type] == None %}
+      {% if config['aliases'][machine_type] == None or not config['aliases'][machine_type] %}
       <p>No devices configurated</p>
```

Comment:
```suggestion
      <p>No devices configured</p>
```

Reply:
cool, wasn't sure if I lacked a sense of humor :)